src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -638,6 +638,10 @@ boolean allowHttps() {
         return true;
     }
 
+    boolean useGraalVMJavaStaticLibraries() {
+        return false;
```

Comment:
might be better to revert this, as the intention is to always use the Java static libs from GraalVM, so we better assume `true` and override with false for the cases where we can't use the static libs yet.